packages/common/src/createKmsAccount.ts

```diff
@@ -0,0 +1,54 @@
+import { KMSClient } from "@aws-sdk/client-kms";
+import { LocalAccount, hashMessage, hashTypedData, keccak256, serializeTransaction } from "viem";
+import { toAccount } from "viem/accounts";
+import { getEthAddressFromKMS } from "./account/kms";
+import { sign } from "./account/sign";
+
+/**
+ * @description Creates an Account from a KMS key and instance.
+ *
+ * @returns A Local Account.
+ */
+export async function createKmsAccount(keyId: string, kmsInstance: KMSClient): Promise<LocalAccount> {
```

Comment:
would be cool to upstream this and he corresponding utils to viem!

Reply:
```suggestion
type CreateKmsAccountOptions = {
  keyId: string;
  client?: KMSClient;
};
export async function createKmsAccount({ keyId, client = new KMSClient() }: CreateKmsAccountOptions): Promise<LocalAccount> {
```

Reply:
also wonder if we should extend `LocalAccount` with a custom `source` like `aws-kms`

Reply:
done! 

what's a `source`? I couldn't find anything on viem

Reply:
it's an argument in account object and `LocalAccount` type

Reply:
I think we can do

```ts
type KMSAccount = LocalAccount<'aws-kms'>;

toAccount({
  source: 'aws-kms',
  ...
});
```

Reply:
Slightly different format but yeah we can do it

https://github.com/wevm/viem/blob/cfe6dee1e259f93d054ea77b18d1f4bf19a0592c/src/accounts/hdKeyToAccount.ts#L30-L34